src/dbus/server/dbus_thread_object.cpp

```diff
@@ -1407,6 +1410,34 @@ otError DBusThreadObject::GetDnssdCountersHandler(DBusMessageIter &aIter)
 #endif // OTBR_ENABLE_DNSSD_DISCOVERY_PROXY
 }
 
+otError DBusThreadObject::GetTrelInfoHandler(DBusMessageIter &aIter)
+{
+#if OTBR_ENABLE_TREL
+    auto           threadHelper = mNcp->GetThreadHelper();
+    auto           instance     = threadHelper->GetInstance();
```

Comment:
nit: may be unnecessary to declare `threadHelper`

Reply:
Done.